site/docs/v1/tech/email-templates/configure-email.adoc

```diff
@@ -47,12 +47,18 @@ Generally speaking, if using port `25` you will select `None`, if using port of
 * `SSL`
 * `TLS`
 
+[field]#Default from address# [optional]#Optional#::
+The default From Address used in sending emails when a from address is not provided for an individual email template. This is an email address ( i.e. **jared@piedpiper.com**).
+
 [field]#Default from name# [optional]#Optional#::
 The default From Name used in sending emails when a from name is not provided on an individual email template. This is the display name part of the email address ( i.e. **Jared Dunn**  <\jared@piedpiper.com>).
 
 [field]#Additional headers# [optional]#Optional# [since]#Available since 1.32.0#::
 One or more line separated SMTP headers to be added to each outgoing email. The header name and value should be separated by an equals sign. ( i.e. `X-SES-CONFIGURATION-SET=Value`).
 
+[field]#Debug enabled# [optional]#Optional#::
```

Comment:
When was this added? We like to add the 'available since' here.

The easiest way I've found is to look at the fusionauth-java-client domain object (starting with Tenant and then going to EmailConfiguration) and then do a blame. https://github.com/FusionAuth/fusionauth-java-client/blame/master/src/main/java/io/fusionauth/domain/EmailConfiguration.java#L37

Then I double check in the release notes and look for the field. Please see if you can figure it out from that.

Reply:
I may need a little help here.  I found https://fusionauth.io/docs/v1/tech/archive/release-notes#version-1-6-0 that talks about a new feature to log SMTP transport errors to the Event Log but don't think that is it.

Reply:
Ok, added the availability date for debug.  I took a look at the Default From Name and Default From Address.  Looks like that came around 4/2020 around release 1.16.  release notes don't go that far back so leaving out.

Reply:
That makes sense to leave them out.